sdk/agrifood/Azure.ResourceManager.AgFoodPlatform/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Release History
 
-## 1.0.0-beta.1 (Unreleased)
+## 1.0.0-beta.1 (2022-09-23)
+
+### Breaking Changes
+
+New design of track 2 initial commit.
+
+### Package Name
+
+The package name has been changed from `Microsoft.Azure.Management.AgFoodPlatform` to `Azure.ResourceManager.AgFoodPlatform`.
```

Comment:
It's a new RP and I don't see it has track 1 package, so we don't need this section to call out the package name change.